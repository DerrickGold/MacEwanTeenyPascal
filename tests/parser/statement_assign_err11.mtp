begin
	(*Missing id*)
	:= 10;
end
.