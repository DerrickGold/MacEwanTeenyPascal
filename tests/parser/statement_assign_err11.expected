Syntax Error: line 3:
Unexpected token (TOK_ASSIGN, ':='), likely missing a(n)  
  'TOK_SEMICOLON' or
Possibly due to malformed or missing:
  Non-terminal: 'Variable'
